application/jaxrs-activemq-quarkus/src/main/java/webapp/tier/service/ActiveMqService.java

```diff
@@ -13,32 +13,29 @@
 import javax.jms.ConnectionFactory;
 import javax.jms.JMSConsumer;
 import javax.jms.JMSContext;
-import javax.jms.Message;
 import javax.jms.Session;
-import javax.jms.TextMessage;
 
 import org.eclipse.microprofile.config.inject.ConfigProperty;
 
 import io.quarkus.runtime.ShutdownEvent;
 import io.quarkus.runtime.StartupEvent;
 import webapp.tier.bean.MsgBean;
 import webapp.tier.interfaces.Messaging;
-import webapp.tier.service.socket.ActiveMqSocket;
 import webapp.tier.util.CreateId;
 import webapp.tier.util.MsgUtils;
 
 @ApplicationScoped
 public class ActiveMqService implements Messaging, Runnable {
 
 	@Inject
-	ConnectionFactory connectionFactory;
+	private ConnectionFactory connectionFactory;
 
 	@Inject
-	ActiveMqSocket amqsock;
+	private ActiveMqConsumer amqconsumer;
 
 	@ConfigProperty(name = "common.message")
 	String message;
-	@ConfigProperty(name = "activemq.splitkey")
+	@ConfigProperty(name = "activemq.split.key")
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6118916)